### PR TITLE
Visible bot paths

### PR DESCRIPTION
**Pull request by Cruix**
Playing as an AI is now one step closer to playing an RTS

untitled

If there is a Pai in the bot, they will see the astar calculated shortest path to the location for 30 seconds, after which they loose the all-access that they get from an AI summon.

:cl:
rscadd: AIs can now activate their diagnostic hud to see the paths that bots will take when patrolling or when summoned.
fix: Player controlled bots now lose their all access from an AI summon after 30 seconds instead of keeping it forever.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
while you're at it can you remove pull verb from pAIbots if that wasn't already done
that might be out of the scope of the pr tho

**Comment by Kmc2000** [comment]
hooooooooo


That looks cool

File: code/modules/mob/living/silicon/silicon.dm [hunk 1]
```diff
@@ -33,17 +33,17 @@
 	var/multihud = FALSE //If you can use more than one hud at the same time
 	var/med_hud = DATA_HUD_MEDICAL_ADVANCED //Determines the med hud to use
 	var/sec_hud = DATA_HUD_SECURITY_ADVANCED //Determines the sec hud to use
-	var/d_hud = DATA_HUD_DIAGNOSTIC //There is only one kind of diag hud
+	var/d_hud = DATA_HUD_DIAGNOSTIC
```
Comment (ShadowDeath6) [inline code/modules/mob/living/silicon/silicon.dm:36]: this kind of change always makes me laugh for some reason

Comment (Cruix) [reply code/modules/mob/living/silicon/silicon.dm:36]: Gotta make the comments accurate or it won't compile

**Comment by Cruix** [comment]
Not pulling thing can be a separate PR because my computer is already in sleep mode at this point

**Comment by matskuman5** [comment]
cool!

**Pull request merged**